packages/aws-serverless/package.json

```diff
@@ -33,12 +33,12 @@
     },
     "./import": {
       "import": {
-        "default": "./build/import-hook.mjs"
+        "default": "./import-hook.mjs"
```

Comment:
I don't think this should be changed in this way. Locally, the two files are emitted into `build`. So instead of removing the path here, we need to adjust the `prepack.ts` script 😬